vscode-trace-extension/src/trace-explorer/time-range/trace-explorer-time-range-data-webview-provider.ts

```diff
@@ -20,6 +20,10 @@ export class TraceExplorerTimeRangeDataProvider implements vscode.WebviewViewPro
         this._experimentDataMap = new TimeRangeDataMap();
     }
 
+    public rerender() {
```

Comment:
I think you can call the lines below in method `updateTraceServerUrl()`. With this you can avoid adding this public API.